src/Settings.hs

```diff
@@ -62,6 +61,13 @@ data AppSettings = AppSettings
     -- to be uploaded.
     }
 
+#if DEVELOPMENT
+#define DEV_BOOL True
+#else
+#define DEV_BOOL False
+#endif
+mkEmbeddedStatic DEV_BOOL "eStatic" [embedDir "static"]
```

Comment:
I'd rather keep the amount of CPP as low as possible. This looks fine but if we are adding it, then I think I'd rather modify the definition of `isDevelopment` to use it too.